LICENSE

```diff
@@ -0,0 +1,2 @@
+YEAR: 2017
+COPYRIGHT HOLDER: Huw Campbell
```

Comment:
This is apparently how it's done in R, with the DESCRIPTION file documenting which license it is.